stable/home-assistant/templates/deployment.yaml

```diff
@@ -53,6 +74,13 @@ spec:
             failureThreshold: 5
             timeoutSeconds: 5
           env:
+            {{- if .Values.configurator.hassApiPassword }}
```

Comment:
Is there a reason we need to now set the `HC_HASS_API_PASSWORD` env variable inside the home-assistant container in addition to the configuration container?

Reply:
Actually I have no idea. I guess this is not really needed. Do you want me to delete it?

Reply:
Sure, go for it!  It may have accidentally been introduced as part of the change.